File: app/src/main/java/wangdaye/com/geometricweather/main/ui/adapter/daily/model/DailyPollen.java
package wangdaye.com.geometricweather.main.ui.adapter.daily.model;

import wangdaye.com.geometricweather.basic.model.weather.Pollen;
import wangdaye.com.geometricweather.main.ui.adapter.daily.DailyWeatherAdapter;

public class DailyPollen implements DailyWeatherAdapter.ViewModel {

    private Pollen pollen;

    public DailyPollen(Pollen pollen) {
        this.pollen = pollen;
    }

    public Pollen getPollen() {
        return pollen;
    }

    public void setPollen(Pollen pollen) {
        this.pollen = pollen;
    }

    public static boolean isCode(int code) {
        return code == 6;
    }

    @Override
    public int getCode() {
        return 6;
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/model/DailyPollen.kt
package org.breezyweather.daily.adapter.model

import org.breezyweather.common.basic.models.weather.Allergen
import org.breezyweather.daily.adapter.DailyWeatherAdapter

class DailyPollen(
    val allergen: Allergen
) : DailyWeatherAdapter.ViewModel {
    override val code = 6

    companion object {
        fun isCode(code: Int) = (code == 6)
    }
}
